src/Nri/Ui/AssignmentIcon/V2.elm

```diff
@@ -0,0 +1,595 @@
+module Nri.Ui.AssignmentIcon.V2 exposing
+    ( diagnostic, planningDiagnosticCircled, unitDiagnosticCircled
+    , practice, practiceCircled
+    , quiz, quizCircled, passageQuizCircled
+    , quickWrite, guidedDraft, peerReview, selfReview
+    , quickWriteCircled, guidedDraftCircled, peerReviewCircled, selfReviewCircled
+    , submitting, rating, revising
+    , startPrimary, startSecondary
+    , home, library, assignArrow
```

Comment:
Can we move these three icons (home, library, and assignArrow) to UiIcon?

Reply:
Done!